llvm/lib/Target/NVPTX/NVPTXTargetMachine.cpp

```diff
@@ -380,7 +380,7 @@ void NVPTXPassConfig::addIRPasses() {
     addStraightLineScalarOptimizationPasses();
   }
 
-  addPass(createAtomicExpandPass());
```

Comment:
we probably should just put this in the standard pass pipeline instead of forcing every target to add this

Reply:
What I understand is I need to add this to the TargetPassconfig::addIRPasses() function which is called by every target. Could you please confirm if this is ryt ? There doesn't seem to be a default pipeline for codegen passes in the new PM. Please do correct me if I am wrong.

Reply:
it's a separate change from this, not related